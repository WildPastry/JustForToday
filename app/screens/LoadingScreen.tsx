import { FontDisplay } from '../components/styles/StyledText';
import { StyleSheet } from 'react-native';

const LoadingScreen: React.FC = (): JSX.Element => {
  // Loading quote
  const selectLoadingQuote = (): string => {
    return 'Loading';
  };

  return <FontDisplay style={styles.title}>{selectLoadingQuote()}</FontDisplay>;
};

const styles = StyleSheet.create({
  title: {
    fontSize: 20
  }
});

export default LoadingScreen;
