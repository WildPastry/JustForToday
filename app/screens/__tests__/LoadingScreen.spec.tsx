import LoadingScreen from '../LoadingScreen';
import renderer from 'react-test-renderer';

describe('<LoadingScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LoadingScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
