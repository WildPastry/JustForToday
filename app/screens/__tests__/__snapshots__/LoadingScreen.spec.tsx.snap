// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingScreen /> renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        {
          "fontSize": 20,
        },
        {
          "fontFamily": "font-display",
        },
      ],
    ]
  }
>
  Loading
</Text>
`;
